lib/rubocop/cop/style/case_equality.rb

```diff
@@ -29,13 +29,20 @@ module Style
       #   (1..100).include?(7)
       #   some_string =~ /something/
       #
-      class CaseEquality < Cop
+      class CaseEquality < Base
```

Comment:
I think this should not be changed:

```suggestion
      class CaseEquality < Cop
```

Reply:
The auto-correction has been added by the new API, I think it is okay to extend `Base`. Rather, changing the inheritance to `Cop` will fail tests.

Reply:
@koic I might be a little confused between Base and Cop. I see most cops inherit from `Cop`, do not `extend AutoCorrector` and have separate method `autocorrect`. Which of these is the preferred/new approach?

Reply:
Inheriting from  `Base` is preferred/new approach. Here's the doc: https://docs.rubocop.org/rubocop/v1_upgrade_notes.html